commit 422ca0551d119315cb50c98e8f94d2c0a5f435e8 (refs/changes/77/5677/2)
Author: Bryan Cantrill <bryan@joyent.com>
Date:   2019-03-01T01:33:49+00:00 (7 months ago)
    
    OS-7631 mdb tests assume advanced "case-sensitive" filesystem technology
    Reviewed by: Jordan Hendricks <jordan.hendricks@joyent.com>
    Approved by: Robert Mustacchi <rm@joyent.com>

diff --git a/usr/src/cmd/mdb/test/format/tst.format-E.mdb b/usr/src/cmd/mdb/test/format/tst.format-cap-E.mdb
similarity index 100%
rename from usr/src/cmd/mdb/test/format/tst.format-E.mdb
rename to usr/src/cmd/mdb/test/format/tst.format-cap-E.mdb
diff --git a/usr/src/cmd/mdb/test/format/tst.format-E.mdb.out b/usr/src/cmd/mdb/test/format/tst.format-cap-E.mdb.out
similarity index 100%
rename from usr/src/cmd/mdb/test/format/tst.format-E.mdb.out
rename to usr/src/cmd/mdb/test/format/tst.format-cap-E.mdb.out
diff --git a/usr/src/cmd/mdb/test/format/tst.format-G.mdb b/usr/src/cmd/mdb/test/format/tst.format-cap-G.mdb
similarity index 100%
rename from usr/src/cmd/mdb/test/format/tst.format-G.mdb
rename to usr/src/cmd/mdb/test/format/tst.format-cap-G.mdb
diff --git a/usr/src/cmd/mdb/test/format/tst.format-G.mdb.out b/usr/src/cmd/mdb/test/format/tst.format-cap-G.mdb.out
similarity index 100%
rename from usr/src/cmd/mdb/test/format/tst.format-G.mdb.out
rename to usr/src/cmd/mdb/test/format/tst.format-cap-G.mdb.out
diff --git a/usr/src/cmd/mdb/test/format/tst.format-J.mdb b/usr/src/cmd/mdb/test/format/tst.format-cap-J.mdb
similarity index 100%
rename from usr/src/cmd/mdb/test/format/tst.format-J.mdb
rename to usr/src/cmd/mdb/test/format/tst.format-cap-J.mdb
diff --git a/usr/src/cmd/mdb/test/format/tst.format-J.mdb.out b/usr/src/cmd/mdb/test/format/tst.format-cap-J.mdb.out
similarity index 100%
rename from usr/src/cmd/mdb/test/format/tst.format-J.mdb.out
rename to usr/src/cmd/mdb/test/format/tst.format-cap-J.mdb.out
diff --git a/usr/src/cmd/mdb/test/format/tst.format-P.mdb b/usr/src/cmd/mdb/test/format/tst.format-cap-P.mdb
similarity index 100%
rename from usr/src/cmd/mdb/test/format/tst.format-P.mdb
rename to usr/src/cmd/mdb/test/format/tst.format-cap-P.mdb
diff --git a/usr/src/cmd/mdb/test/format/tst.format-P.mdb.out b/usr/src/cmd/mdb/test/format/tst.format-cap-P.mdb.out
similarity index 100%
rename from usr/src/cmd/mdb/test/format/tst.format-P.mdb.out
rename to usr/src/cmd/mdb/test/format/tst.format-cap-P.mdb.out
diff --git a/usr/src/cmd/mdb/test/format/tst.format-R.mdb b/usr/src/cmd/mdb/test/format/tst.format-cap-R.mdb
similarity index 100%
rename from usr/src/cmd/mdb/test/format/tst.format-R.mdb
rename to usr/src/cmd/mdb/test/format/tst.format-cap-R.mdb
diff --git a/usr/src/cmd/mdb/test/format/tst.format-R.mdb.out b/usr/src/cmd/mdb/test/format/tst.format-cap-R.mdb.out
similarity index 100%
rename from usr/src/cmd/mdb/test/format/tst.format-R.mdb.out
rename to usr/src/cmd/mdb/test/format/tst.format-cap-R.mdb.out
